Cross Region VM Restore Points/Cross Region Creation of VM Restore Points.md

```diff
@@ -0,0 +1,457 @@
+                        },
+                        "diskRestorePoint": {
+                            "id": "/subscriptions/<subId>/resourceGroups/<rgName>/restorePointCollections/<rpcName>/restorePoints/<rpName>/diskRestorePoints/<diskRestorePointName>"
+                        }
+                    }
+                ]
+            },
+            "diagnosticsProfile": {
+                "bootDiagnostics": {
+                    "enabled": true,
+                    "storageUri": " http://storageaccount.blob.core.windows.net/"
+                }
+            }
+        },
+"optionalProperties": "opaque bag of properties to be passed to extension",
+        "consistencyMode": "CrashConsistent | FileSystemConsistent | ApplicationConsistent",
+        "provisioningState": "Succeeded | Failed | Creating | Deleting",
+        "provisioningDetails": {
+            "creationTime": "Creation Time of Restore point in UTC",
+            "totalUsedSizeInBytes": "25",
+            "statusCode": "status code reported by the extension",
+            "statusMessage": "status message reported by the extension"
+        },
+        "instanceView": {
+            "statuses": [
+                {
+                    "code": "ReplicationState/succeeded",
+                    "level": "Info",
+                    "displayStatus": "Replication succeeded"
+                }
+            ],
+            "diskRestorePoints": [
+                {
+                    "id": "<diskRestorePoint Arm Id>",
+                    "replicationStatus": {
+                        "status": {
+                            "code": "ReplicationState/succeeded",
+                            "level": "Info",
+                            "displayStatus": "Replication succeeded"
+                        },
+                        "completionPercent": "<completion percentage of the replication>"
+                    }
+                }
+            ]
+        }
+    }
+}
+```
+
+## Known issues and limitations
+* Cross-region copy functionality is only supported via REST APIs. ARM Template deployment, CLI, SDK, PS will be added later. 
+* Concurrent creation of Restore Points for a VM in the same or different regions is not supported
+* Error messages may not reflect the actual error that has occurred. Error messages will be updated.
+* Restore point of CMK encrypted VM and disks will be encrypted using PMK in the target region. Customer needs to use CMK when restoring the disks and VM from the restore point.
+* Currently, the replication progress in only updated once every 10mins. Hence for disks that have low churn, there can be scenarios where only the initial (0) and the terminal replication progress (100) can be seen.
+* If there is huge amount data to be copied to the target region, depending on the bandwidth available between the regions, the creation time could be couple of days. If the creation time exceeds 2 weeks, the restore points creation operation will be terminated.
```

Comment:
We are talking about "creation time"... Isn't it rather "copy time"?

Reply:
You are right. changed it.